src/System.Runtime.CompilerServices.Unsafe/src/System.Runtime.CompilerServices.Unsafe.ilproj

```diff
@@ -0,0 +1,94 @@
+<Project Sdk="Microsoft.NET.Sdk.IL">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(NetFrameworkMinimum);netstandard2.0;net7.0</TargetFrameworks>
+    <StrongNameKeyId>Microsoft</StrongNameKeyId>
+    <IsPackable>false</IsPackable>
+    <!-- Make sure that the DebuggableAttribute is set properly. -->
+    <DebugOptimization>IMPL</DebugOptimization>
+    <DebugOptimization Condition="'$(Configuration)' == 'Release'">OPT</DebugOptimization>
+    <!-- ILASM will always produce a portable PDB -->
+    <DebugType>portable</DebugType>
+    <CoreCompileDependsOn>$(CoreCompileDependsOn);GenerateVersionFile</CoreCompileDependsOn>
+    <DocumentationFile>$(MSBuildThisFileDirectory)System.Runtime.CompilerServices.Unsafe.xml</DocumentationFile>
+    <IlasmFlags>$(IlasmFlags) -DEBUG=$(DebugOptimization)</IlasmFlags>
+    <PackageDescription>Provides the System.Runtime.CompilerServices.Unsafe class, which provides generic, low-level functionality for manipulating pointers.
+
+Commonly Used Types:
+System.Runtime.CompilerServices.Unsafe</PackageDescription>
+    <IsPlaceholderTargetFramework Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETCoreApp'">true</IsPlaceholderTargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+    <CoreAssembly>netstandard</CoreAssembly>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(TargetFramework)' == '$(NetFrameworkMinimum)'">
+    <CoreAssembly>mscorlib</CoreAssembly>
+    <ExtraMacros>#define $(TargetFramework)</ExtraMacros>
```

Comment:
I'm struggling to find any place that uses this define.

Reply:
The only macro I found in the *.il file is `netcoreapp`

Reply:
Were you looking in other places also?

Reply:
Actually, if I remove it, the *.il fails to build. I need to keep it. This is the error:

```

C:\Users\calope\source\repos\maintenance-packages\artifacts\obj\System.Runtime.CompilerServices.Unsafe\Debug\net462\version\version.h(9): error : syntax error at token '{' in: { [C:\Users\calope\source\re 
pos\maintenance-packages\src\System.Runtime.CompilerServices.Unsafe\src\System.Runtime.CompilerServices.Unsafe.ilproj::TargetFramework=net462]
C:\.tools\.nuget\packages\microsoft.net.sdk.il\8.0.0\targets\Microsoft.NET.Sdk.IL.targets(136,5): error MSB3073: The command ""C:\.tools\.nuget\packages\\runtime.win-x64.microsoft.netcore.ilasm\6.0.0\runt 
imes\win-x64\native/ilasm" -QUIET -NOLOGO -DLL  -DEBUG=IMPL -INCLUDE="C:\Users\calope\source\repos\maintenance-packages\artifacts\obj\System.Runtime.CompilerServices.Unsafe\Debug\net462\version" -OUTPUT=" 
C:\Users\calope\source\repos\maintenance-packages\artifacts\obj\System.Runtime.CompilerServices.Unsafe\Debug\net462\System.Runtime.CompilerServices.Unsafe.dll" -KEY="C:\.tools\.nuget\packages\microsoft.do 
tnet.arcade.sdk\8.0.0-beta.24360.5\tools\snk/MSFT.snk" System.Runtime.CompilerServices.Unsafe.il" exited with code 1. [C:\Users\calope\source\repos\maintenance-packages\src\System.Runtime.CompilerServices 
.Unsafe\src\System.Runtime.CompilerServices.Unsafe.ilproj::TargetFramework=net462]
```

That version.h file is generated with the define like this:
```c
#define CORE_ASSEMBLY "mscorlib"
#define ASSEMBLY_VERSION "1:0:0:0"
#define CORE_ASSEMBLY_VERSION "4:0:0:0"
#define FILE_VERSION "{string('1.0.24.36201')}"
#define INFORMATIONAL_VERSION "{string('1.0.0-dev.24362.1')}"
#define net462
// Metadata version: v4.0.30319
.assembly extern CORE_ASSEMBLY
{
  .publickeytoken = (B7 7A 5C 56 19 34 E0 89 )
  .ver CORE_ASSEMBLY_VERSION
}
```